manifests/init.pp

```diff
@@ -102,8 +105,9 @@
   Stdlib::Port $api_port = 24817,
   Stdlib::Host $content_host = '127.0.0.1',
   Stdlib::Port $content_port = 24816,
-  Stdlib::Absolutepath $webserver_static_dir = '/var/lib/pulp/docroot',
-  Stdlib::Absolutepath $pulp_static_root = '/var/lib/pulp/assets',
+  Stdlib::Absolutepath $apache_docroot = $user_home,
```

Comment:
Setting the docroot to the user home feels like a bad idea, but I guess it matches what Pulp does.